pkg/rancher-desktop/pages/containers/logs/_id.vue

```diff
@@ -505,6 +534,11 @@ export default defineComponent({
   overflow: hidden;
   display: flex;
   flex-direction: column;
+  opacity: 1;
+
+  &.terminal-hidden {
+    opacity: 0;
```

Comment:
I think browsers have special-cased it so performance doesn't actually matter anymore, but wouldn't saying `visibility: hidden` be more obvious?